src/components/LeftSidebar/SearchBox/SearchBox.vue

```diff
@@ -170,6 +171,9 @@ export default {
 .search-box {
 	:deep(.input-field__input) {
 		border-radius: var(--border-radius-pill);
+		&:focus, &:hover {
```

Comment:
```suggestion
		&:focus,
		&:focus-visible,
		&:focus-within,
		&:hover {
```
Not sure if needed, but might worth a check with focus on trailing button and keyboard navigation

Reply:
`focus-within` is not needed, there are no elements "within" `<input>`.

For `focus-visible` we have only `unset` for browser styles. When input is on focus, it has `:focus` state anyway.